doc/classes/ProjectSettings.xml

```diff
@@ -1118,6 +1118,10 @@
 		<member name="layer_names/3d_render/layer_9" type="String" setter="" getter="" default="&quot;&quot;">
 			Optional name for the 3D render layer 9. If left empty, the layer will display as "Layer 9".
 		</member>
+		<member name="logging/buffered_logging/buffer_size" type="int" setter="" getter="" default="100">
+		</member>
+		<member name="logging/buffered_logging/enable_buffered_logging" type="bool" setter="" getter="" default="false">
+		</member>
```

Comment:
```suggestion
		<member name="logging/buffered_logging/enable_buffered_logging" type="bool" setter="" getter="" default="false">
			If [code]true[/code], allows retrieving standard output and error logging streams using [method Engine.get_log_buffer], [method Engine.get_error_buffer], [signal Engine.log_message], and [signal Engine.error_message]. For example, this can be used to display built-in messages in an in-game console.
		</member>
```